tests/test_process.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+from xarray import open_dataset
+from tempfile import NamedTemporaryFile
+from orca.process import process_request, build_url
+from orca.finder import find_filepath
+
+
+@pytest.mark.parametrize(
+    ("filepath", "variable", "lat", "lon"),
+    [
+        (
+            "/storage/data/climate/downscale/BCCAQ2/bccaqv2_with_metadata/tasmax_day_BCCAQv2+ANUSPLIN300_CSIRO-Mk3-6-0_historical+rcp85_r1i1p1_19500101-21001231.nc",
+            "tasmax[0:1:55114]",
+            "[91:91]",
+            "[206:206]",
+        ),
+        (
+            "/storage/data/climate/downscale/BCCAQ2/bccaqv2_with_metadata/tasmax_day_BCCAQv2+ANUSPLIN300_CSIRO-Mk3-6-0_historical+rcp85_r1i1p1_19500101-21001231.nc",
+            "tasmax[0:55114]",
+            "[91:91]",
+            "[206:206]",
+        ),
```

Comment:
Between the cases there is only one item that is changing. You can parametrize the items separately to avoid repeating parameters. See the example in the [docs](https://docs.pytest.org/en/stable/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions) (final example in the linked section).